package.json

```diff
@@ -10,9 +10,11 @@
     "preview": "vite preview"
```

Comment:
### Optional
- [ ] We are advised to use `react` as our script , however, we have `vite` in our script. Going forward, your project can be invalidated because of that. Kindly take note and change it accordingly